test/Test/Lynx/Form.purs

```diff
@@ -201,51 +209,74 @@ testPageEither :: Either String (Page Expr)
 testPageEither = decodeJson =<< jsonParser testPageJson
 
 testPageJson :: String
-testPageJson = """
+testPageJson =
+  """
   { "name": "Profile"
   , "tabs":
-    [ """ <> testTab <> """
+    [ """
+    <> testTab
+    <> """
     ]
   }
 """
 
 testTab :: String
-testTab = """
+testTab =
+  """
   { "name": "User"
   , "link": "user"
   , "sections":
-    [ """ <> testSection <> """
-    , """ <> testSequence <> """
+    [ """
+    <> testSection
+    <> """
+    , """
+    <> testSequence
+    <> """
     ]
   }
 """
 
 testSection :: String
-testSection = """
+testSection =
+  """
   { "type": "TabSection"
   , "name": "Name"
   , "fields":
-    [ """ <> firstName <> """
-    , """ <> lastName <> """
-    , """ <> active <> """
+    [ """
+    <> firstName
+    <> """
+    , """
+    <> lastName
+    <> """
+    , """
+    <> active
+    <> """
```

Comment:
I guess it's not a big deal, but is this something we could configure? All these JSON strings are a bit less readable now

Reply:
What did you have in mind?